tools/sandbox_e2e_smoke_test.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import annotations
+import sys
+import multiprocessing
+from pathlib import Path
+
+project_root = Path(__file__).parent.parent
+for python_path in [str(project_root), str(project_root / "src")]:
+    sys.path.append(python_path)
+
+
+def launch_sandbox_script():
+    from sandbox.basic import app as basic_app
+
+    basic_app.run()
+
+
+process = multiprocessing.Process(target=launch_sandbox_script, daemon=True)
+process.start()
+
+process.join(timeout=2)
```

Comment:
A very simple script, that could probably be done in Bash too but why not using Python? ^^ 
 - Start the "basic" sandbox script in a Python subprocess, thanks to the nice [multiprocessing](https://devdocs.io/python~3.10/library/multiprocessing) module 
 - Let it do its things for at most 2 seconds
 - Check if it crashed. 
    - If yes, we exit with an error code.
    - If not, we kill it (sorry, BasicApp! :bow: )  and exit with a success `0` code